styles/prosilver/template/pastebin_body.html

```diff
@@ -88,5 +66,27 @@
 </div>
 
 
+<!-- IF S_AUTH_VIEW -->
+<div class="mini-panel navbar">
+	<span class="corners-top"><span></span></span>
```

Comment:
Wir können den Code auch in einem gesondertem PR entfernen. Dann ist das thematisch getrennt und du musst das hier nicht auch noch machen.

Ich überlasse dir mal die Entscheidung, ob du das mit machst oder übrig lässt.

Grüße